### ISSUE TITLE
Dependency of nuget package

### ISSUE THREAD

**Issue by ivarne** [Altinn/codelists-lib-dotnet#9]
### Description of the bug

`Altinn.CodeLists.csproj` references that this project should work with any `7.` version of `Altinn.App.Core`

https://github.com/Altinn/codelists-lib-dotnet/blob/caf9dae533b5c306e2f8d265314f4c6c7088d7ec/src/Altinn.Codelists/Altinn.Codelists.csproj#L26

This works when referencing CodeLists using `<ProjectReference`, but when the nuget package gets built it gets built with the newest prerelease of `Altinn.App.Core`, it seems to require all users of the package to also be on the latest prerelease.

```xml
    <PackageReference Include="Altinn.App.Api" Version="7.8.0">
      <CopyToOutputDirectory>lib\$(TargetFramework)\*.xml</CopyToOutputDirectory>
    </PackageReference>
    <PackageReference Include="Altinn.App.Core" Version="7.8.0" />
    <PackageReference Include="Altinn.Codelists" Version="0.5.0-preview.9" />
```
results in

![image](https://user-images.githubusercontent.com/131616/236135424-f594453b-1c03-4636-b79a-2cc9a8e47e4f.png)

Another comment is that after https://github.com/Altinn/codelists-lib-dotnet/commit/caf9dae533b5c306e2f8d265314f4c6c7088d7ec used the `Description` field that was introduced in [`Altinn.App.Core v7.8.0`](https://github.com/Altinn/app-lib-dotnet/releases/tag/v7.8.0), so the requirement should probably be updated.

### Steps To Reproduce

ddd

**Comment by benteSSB** [comment]
org/ssb

**Comment by RonnyB71** [comment]
Closing this. Fixed in version 0.5.1 of the package.
